src/Ormolu/Printer/Meat/Declaration/Rule.hs

```diff
@@ -24,7 +24,7 @@ p_ruleDecls = \case
 
 p_ruleDecl :: RuleDecl GhcPs -> R ()
 p_ruleDecl = \case
-  HsRule NoExt ruleName activation ruleBndrs lhs rhs -> do
+  HsRule NoExt ruleName activation _ ruleBndrs lhs rhs -> do
```

Comment:
Here we are skipping `rd_tyvs :: Maybe [LHsTyVarBndr (NoGhcTc pass)]`. I will try to find an example where `rd_tyvs` is non-empty.

Reply:
Probably this:

```
{-# RULES "example" forall a. forall (x :: a). id x = x #-}
```